corehq/apps/integration/kyc/views.py

```diff
@@ -153,6 +157,19 @@ def _report_verification_status_metric(self, success_count, failure_count):
             tags={'domain': self.request.domain}
         )
 
+    def _report_success_on_reverification_metric(self, existing_failed_user_ids, results):
+        successful_user_ids = [user_id for user_id, status in results.items() if status is True]
+        reverification_success_count = len(set(existing_failed_user_ids) & set(successful_user_ids))
+        if reverification_success_count > 0:
+            metrics_counter(
+                'commcare.integration.kyc.reverification.success.count',
+                reverification_success_count,
+                tags={'domain': self.request.domain}
+            )
+
+    def _get_existing_failed_users(self, kyc_users):
+        return [kyc_user.user_id for kyc_user in kyc_users if kyc_user.kyc_is_verified is False]
```

Comment:
Nit: Same as previous, the `is False` could be dropped here.
```suggestion
        return [kyc_user.user_id for kyc_user in kyc_users if not kyc_user.kyc_is_verified]
```

Reply:
Used a choice class. See https://github.com/dimagi/commcare-hq/commit/68b3091cd589f814b0af6bf68a878ce588792d69